next-env.d.ts

```diff
@@ -1,3 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
Did you add this or, was it also generated?

Reply:
it was generated - i was also surprised by this

Reply:
Probably helpful regardless 😄 I was surprised when it started getting unconditionally overwritten with Next 11.